images/openemr/openemr.conf

```diff
@@ -42,14 +45,3 @@ CustomLog ${APACHE_LOG_DIR}/access.log combined
     #RewriteCond %{HTTPS} off
     #RewriteRule (.*) https://%{HTTP_HOST}/$1 [R,L]
 </VirtualHost>
-<VirtualHost _default_:8443>
```

Comment:
Is this VirtualHost necessary? There is a separate ssl.conf which is copied into /etc/httpd/conf.d. Ideally we could encapsulate all of the SSL settings/config into ssl.conf unless there is a compelling reason not too. I'm not an Apache dev/admin so additional info here is welcome.